TeslaSUI/TeslaSUI/Views/BottomSheetView.swift

```diff
@@ -0,0 +1,74 @@
+// BottomSheetView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Нижний список
+struct BottomSheetView<Content: View>: View {
+    let content: () -> Content
+    
+    @GestureState private var gestureOffset = CGSize.zero
+    @State private var currentMenuOffsetY: CGFloat = 0.0
+    @State private var lastMenuOffsetY: CGFloat = 0.0
+    
+    var dragGesture: some Gesture {
+        DragGesture()
+            .updating($gestureOffset) { value, state, _ in
+                state = value.translation
+                onChangeMenuOffset()
+            }
+            .onEnded { _ in
+                let maxHeight = UIScreen.main.bounds.height / 6
+                
+                withAnimation {
+                    if -currentMenuOffsetY > maxHeight / 2 {
+                        currentMenuOffsetY = -maxHeight
+                    } else {
+                        currentMenuOffsetY = 0
+                    }
+                    lastMenuOffsetY = currentMenuOffsetY
+                }
+            }
+    }
+    
+    func onChangeMenuOffset() {
```

Comment:
```suggestion

// MARK: - Private Methods

    private func onChangeMenuOffset() {
```
вроде можно сделать приватной и перенести под боди тогда
если нельзя то марка паблик методс